<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MAZHAVIKA" ignored="0" name="TestonChrome" tests="1" failures="1" timestamp="09 Apr 2020 17:03:04 GMT" time="0.084" errors="0">
  <testcase name="@AfterTest close" time="0.0" classname="selenium.practise.basis.TestNGParallaltest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at selenium.practise.basis.TestNGParallaltest.close(TestNGParallaltest.java:82)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest close -->
  <testcase name="function" time="0.011" classname="selenium.practise.basis.TestNGParallaltest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at selenium.practise.basis.TestNGParallaltest.function(TestNGParallaltest.java:59)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- function -->
</testsuite> <!-- TestonChrome -->
